### DISCUSSION TITLE
warn(prisma-client) Already 10 Prisma Clients are actively running with prisma-multi-tenant

### DISCUSSION THREAD

**Discussion by asaadawey** [prisma/prisma#4794] [Q&A] [answered]
Getting this  error while using prisma-multi-tenant addon 
warn(prisma-client) Already 10 Prisma Clients are actively running.

any idea

**Comment by nikolasburk** [comment, marked as answer]
Hey @asaadawey, I'm not deeply familiar with the `prisma-multi-tenant` library so I'm not sure if this behaviour might even be expected. The warning occurs when you have more than 10 active `PrismaClient` instances. With a "normal" (i.e. single-tenant) setup you typically one have a single instance of `PrismaClient` in your application, so in that case the warning is an indication that you might accidentally be instantiating too many instances of `PrismaClient`. In a multi-tenant setup, it might be expected to have multiple `PrismaClient` instances since each could be responsible for a different tenant's DB.

Maybe [ask the same question in the `prisma-multi-tenant` repo](https://github.com/Errorname/prisma-multi-tenant/issues) to get some clarity on how it works under the hood?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
